source/core/src/main/com/csse3200/game/components/DialogComponent.java

```diff
@@ -27,6 +28,8 @@ public class DialogComponent extends RenderComponent {
     private static float textLength = 0f;
     private static final float FRAMES_PER_CHAR = 5f;
     private static final float MAX_WIDTH = 15f;
+    
+    private int cooldownTime = 2;
```

Comment:
What is the use of cooldown time ?

Reply:
Earlier there was a bug due to which dialogue box was getting dismissed if player presses enter before dialogue gets displayed completely, so I did this just to resolve that.